core/server/services/settings/yaml-parser.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const yaml = require('js-yaml'),
+    fs = require('fs-extra'),
+    debug = require('ghost-ignition').debug('settings:yaml-parser'),
+    common = require('../../lib/common');
+
+/**
+ * Takes a YAML file, parses it and returns a JSON Object
+ * @param {String} filePath the absolute path to the file
+ * @param {String} fileName the name of the file incl. extension
+ * @returns {Promise<Object>} parsed
+ * @description Takes a YAML file, parses it and returns a JSON Object.
+ */
+module.exports = function parseYaml(filePath, fileName) {
```

Comment:
The fn is called `parseYaml`, but it also reads a file 🤔 
And the `settingsLoader` only reads the fileNames. Not high prio to change this, but wanted to mention it.
Not saying that it's wrong, just noticed.

Also the fn describes:

> Takes a YAML file, parses it and returns a JSON Object

Reply:
> The fn is called parseYaml, but it also reads a file

Agreed, but it also parses the yaml file after it's accessed.

> And the settingsLoader only reads the fileNames

But the settingsLoader is also a wrapper of the Yaml parser and returns the settings object. 

🤔